src/auth/screens/login.screen.js

```diff
@@ -229,130 +243,120 @@ class Login extends Component {
-                      disabled={this.state.cancelDisabled}
-                      textStyle={styles.buttonText}
-                      onPress={() =>
-                        this.setModalVisible(!this.state.modalVisible)}
-                    />
-                  </View>
-                </View>
-              </Modal>
-              <View style={styles.miniSection}>
-                <Image
-                  style={styles.logo}
-                  source={require('../../assets/logo.png')}
+        {this.shouldDisplayIntro() && (
+          <Swiper
+            loop={false}
+            dotColor="#FFFFFF55"
+            activeDotColor={colors.white}
+          >
+            <View style={[styles.slide, styles.slide1]}>
+              <Image
+                style={styles.logo}
+                source={require('../../assets/logo.png')}
+              />
+              <Text style={styles.title}>
+                {translate('auth.login.welcomeTitle', locale)}
+              </Text>
+              <Text style={styles.message}>
+                {translate('auth.login.welcomeMessage', locale)}
+              </Text>
+            </View>
+            <View style={[styles.slide, styles.slide2]}>
+              <Icon
+                style={styles.icon}
+                color={colors.white}
+                size={85}
+                name="bell"
+                type="octicon"
+              />
+              <Text style={styles.title}>
+                {translate('auth.login.notificationsTitle', locale)}
+              </Text>
+              <Text style={styles.message}>
+                {translate('auth.login.notificationsMessage', locale)}
+              </Text>
+            </View>
+            <View style={[styles.slide, styles.slide3]}>
+              <Icon
+                containerStyle={styles.iconMargin}
+                style={styles.icon}
+                color={colors.white}
+                size={85}
+                name="repo"
+                type="octicon"
+              />
+              <Text style={styles.title}>
+                {translate('auth.login.reposTitle', locale)}
+              </Text>
+              <Text style={styles.message}>
+                {translate('auth.login.reposMessage', locale)}
+              </Text>
+            </View>
+            <View style={[styles.slide, styles.slide4]}>
+              <Icon
+                containerStyle={styles.iconMargin}
+                style={styles.icon}
+                color={colors.white}
+                size={85}
+                name="git-pull-request"
+                type="octicon"
+              />
+              <Text style={styles.title}>
+                {translate('auth.login.issuesTitle', locale)}
+              </Text>
+              <Text style={styles.message}>
+                {translate('auth.login.issuesMessage', locale)}
+              </Text>
+            </View>
+          </Swiper>
+        )}
+        {this.shouldDisplayIntro() && (
```

Comment:
Why this check? Can we combine into one?

Reply:
We sadly cannot, `<Swiper>` don't work at all in Android when nested into a view, that's why I need these multiple checks/rendering